desktop/src/main/fileProcessing/workers/utilities/generateMetadataInBatches.ts

```diff
@@ -4,6 +4,13 @@ import { parentPort } from "node:worker_threads";
 import { calculateChecksum } from "./calculateChecksum";
 import { countFiles } from "./countFiles";
 import { formatFileSize } from "./formatFileSize";
+import { exec } from "node:child_process";
+import { promisify } from "node:util";
+
+const execPromise = promisify(exec);
+const { stat, readdir } = fsPromises;
+
+let processedFileCount = 0;
```

Comment:
Duplicated declarations. Copy-paste error or possible merge conflict? smoke test - quick build and startup always help identify these
```suggestion
```